index.php

```diff
@@ -6,266 +6,324 @@
+                    <option value='38186'>Bokutachi wa Benkyou ga Dekinai</option>
+                    <option value='37435'>Carole & Tuesday</option>
+                    <option value='38226'>Chou Kadou Girl ⅙: Amazing Stranger</option>
+                    <option value='38731'>Diamond no Ace: Act II</option>
+                    <option value='39063'>Fairy Gone</option>
+                    <option value='37806'>Gunjou no Magmel</option>
+                    <option value='38091'>Hachigatsu no Cinderella Nine</option>
+                    <option value='38268'>Hangyakusei Million Arthur 2nd Season</option>
+                    <option value='37614'>Hitoribocchi no 笳銀雷 Seikatsu</option>
+                    <option value='38295'>Joshikausei</option>
+                    <option value='36407'>Kenja no Mago</option>
+                    <option value='38000'>Kimetsu no Yaiba</option>
+                    <option value='38080'>Kono Oto Tomare!</option>
+                    <option value='34620'>Kono Yo no Hate de Koi wo Utau Shoujo YU-NO</option>
+                    <option value='37964'>Mayonaka no Occult Koumuin</option>
+                    <option value='38778'>Midara na Ao-chan wa Benkyou ga Dekinai</option>
+                    <option value='38098'>Mix: Meisei Story</option>
+                    <option value='38150'>Namu Amida Butsu!: Rendai Utena</option>
+                    <option value='38397'>Nande Koko ni Sensei ga!?</option>
+                    <option value='38814'>Nobunaga-sensei no Osanazuma</option>
+                    <option value='38707'>RobiHachi</option>
+                    <option value='37426'>Sarazanmai</option>
+                    <option value='38787'>Senryuu Shoujo</option>
+                    <option value='38759'>Sewayaki Kitsune no Senko-san</option>
+                    <option value='37952'>Shoumetsu Toshi</option>
+                    <option value='38004'>World Witches Series: 501-butai Hasshin Shimasu!</option>
+                    <option value='37940'>Yatogame-chan Kansatsu Nikki</option>
+                </select>
+            </div>
+            <br>
+            <br>
+            <div id='code_div' style='display:none'>
+                <hr>
+                <br> Your verification code is:
+                <br>
+                <br><b><code><span id='code'></span></code></b>
+                <br>
+                <br>Please copy and paste this code inside the "Location" field on your MAL profile. Once this is done, click the button below to register your team.
+            </div>
+            <br>
+            <br>
+            <br>
+            <button id='register_btn' style='display:none' class='btn btn-primary btn-block'>Register Team</button> -->
+
+            <div class='text-center'>Registration is now closed.</div>
+            <br>
+            <br>
+            <br>
+
+            <h2 class='text-center' id='subtitle'>Check Your Team</h2>
```

Comment:
Looks really cool. How would you expect this to look when registration isn't closed? Because I'd imagine that's the primary use case here right?

Reply:
I just uncommented the HTML and it would look like the below

![image](https://user-images.githubusercontent.com/24740091/56070355-0dfc1500-5d4d-11e9-89e8-10b36d7106bb.png)

Reply:
Awesome, thanks